doctor_visits/delphi_doctor_visits/config.py

```diff
@@ -21,21 +21,24 @@ class Config:
     CLI_COLS = ["Covid_like", "Flu_like", "Mixed"]
     FLU1_COL = ["Flu1"]
     COUNT_COLS = CLI_COLS + FLU1_COL + ["Denominator"]
-    DATE_COL = "ServiceDate"
-    GEO_COL = "PatCountyFIPS"
-    AGE_COL = "PatAgeGroup"
-    HRR_COLS = ["Pat HRR Name", "Pat HRR ID"]
+    DATE_COL = "ServiceDate" #"servicedate"
+    GEO_COL = "PatCountyFIPS" #"patCountyFIPS"
+    AGE_COL = "PatAgeGroup" #"patAgeGroup"
+    HRR_COLS = ["Pat HRR Name", "Pat HRR ID"]#["patHRRname", "patHRRid"]
```

Comment:
what are these comments for?

Reply:
At some point, HSP changed the column names of the files they were sending us. I was testing the code on a recent drop (with lowercase column names). The code originally had the uppercase names, so I changed them back to pass the tests and to merge the PR.